infra/terraform/test-org/ci-project/cleaner.tf

```diff
@@ -0,0 +1,52 @@
+
+
+resource "google_cloudbuild_trigger" "org_iam_cleaner_trigger" {
+  provider = google-beta
+
+  description = "Reset org IAM"
+  github {
+    owner = local.gh_orgs.infra
+    name  = local.gh_repos.infra
+    # this will be invoked via cloud scheduler, hence using a regex that will not match any branch
+    push {
+      branch = ".^"
+    }
+  }
+
+  filename = "infra/terraform/test-org/org-iam-policy/cloudbuild.yaml"
+}
+
+resource "google_service_account" "service_account" {
+  account_id   = "org-iam-clean-trigger"
+  display_name = "SA used by Cloud Scheduler to trigger cleanup builds"
+}
+
+resource "google_project_iam_member" "project" {
+  role   = "roles/cloudbuild.builds.editor"
```

Comment:
Could we use a custom role? This SA should only need `cloudbuild.builds.create` permission.

Reply:
done, I ended up needing ` "cloudbuild.builds.create", "cloudbuild.builds.get"` since I am calling the `projects.triggers.run` method/